Lib/test/test_concurrent_futures/test_interpreter_pool.py

```diff
@@ -498,6 +499,20 @@ def test_import_interpreter_pool_executor(self):
         self.assertEqual(p.stdout.decode(), '')
         self.assertEqual(p.stderr.decode(), '')
 
+    def test_thread_name_prefix(self):
+        self.assertStartsWith(self.executor._thread_name_prefix,
+                              "InterpreterPoolExecutor-")
+
+    @unittest.skipUnless(hasattr(_thread, '_get_name'), "missing _thread._get_name")
+    def test_thread_name_prefix_with_thread_get_name(self):
+        def f():
```

Comment:
Mostly LGTM

Just nits:

Maybe we can use a more meanful function name like `get_thread_name` or else instance of `f`